app/src/test/kotlin/org/imperial/mrc/modelserver/database/UsersTests.kt

```diff
@@ -0,0 +1,47 @@
+package org.imperial.mrc.modelserver.database
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.pac4j.core.credentials.password.JBCryptPasswordEncoder
+import org.pac4j.sql.profile.DbProfile
+import org.pac4j.sql.profile.service.DbProfileService
+import org.postgresql.jdbc3.Jdbc3PoolingDataSource
+import org.mindrot.jbcrypt.BCrypt
+
+//Tests that pac4j can access the database
```

Comment:
I just wanted to check that the pac4j DbProfileService class could access the table to add and remove users.

Reply:
I think we'll want to use Libsodium, but we can raise a separate ticket for that.